### PR TITLE
Convert Fahrenheit to Celsius

### LINKED ISSUE #470: Convert Fahrenheit to Celsius C++ Code
# Description
This code will convert Fahrenheit temperature to Celsius Code

# Important Checklist 
-   [x] I have read the contribution guidelines before making this issue.
-   [x] I have verified this issue program is not already implemented
-   [x] I have not pushed this code anywhere else (even my repo)

# Languages and Assignees:
-   [x] Python: @Addhyay
-   [x] Java: @AK271 
-   [x] C: @saitaruns 
-   [x] C++: @shubhamsingla807

### PR DESCRIPTION
**Pull request by addhyay**
# Description

Added a Python code which converts Fahrenheit to Celsius

Fixes #470

## Type of change

# Checklist:

-   [x] I have read the contribution guidelines before making this PR.
-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented in my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation (Language folder or any other readme instructed to)
-   [x] My changes generates no new warnings
-   [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by addhyay** [comment]
@kaustubhgupta I'm stuck here, why it is saying CodeFactor - 1 issue found? I did check it, it's with **except** which I used in Multiplicative Inverse algorithm, could you please verify it, because I'm not understanding what to do?

**Comment by kaustubhgupta** [comment]
@Addhyay  Generally, we specify what type of exception we are handling. For instance,
```python

a= 10
b = 'hello'
try:
    print(a+b)
except TypeError as e:
    print(e)
```
will give result as:
```bash
unsupported operand type(s) for +: 'int' and 'str'
```

So wherever you are using except block, specify the type of exception and it should pass the test

**Comment by addhyay** [comment]
> @Addhyay Generally, we specify what type of exception we are handling. For instance,
> 
> ```python
> a= 10
> b = 'hello'
> try:
>     print(a+b)
> except TypeError as e:
>     print(e)
> ```
> 
> will give result as:
> 
> ```shell
> unsupported operand type(s) for +: 'int' and 'str'
> ```
> 
> So wherever you are using except block, specify the type of exception and it should pass the test

Thank you!!

**Comment by kaustubhgupta** [comment]
@Addhyay Secondly I was reviewing this PR and found that you pushed code for both issues in one PR. Please fix that

**Comment by addhyay** [comment]
Okay, I'll make another pull request, Thank you for notifying!

**Comment by addhyay** [comment]
Done !!

File: Python/Fahrenheit_to_celsius.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+fahrenheit = float(input())
```
Comment (kaustubhgupta) [inline Python/Fahrenheit_to_celsius.py]: I know the issue is pretty easy but don't finish it a one-liner. Format the code properly with comments

File: Python/Fahrenheit_to_celsius.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+fahrenheit = float(input())
+print(str((fahrenheit - 32) * 5/9)+'°C')
```
Comment (kaustubhgupta) [inline Python/Fahrenheit_to_celsius.py]: Update python folder readme

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Python/Fahrenheit_to_celsius.py (hunk 1 above)
Comment (addhyay) [reply Python/Fahrenheit_to_celsius.py]: Okay, I'll design a function for it.

File: Python/Fahrenheit_to_celsius.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+def fahrenheit_to_celsius(start_temperature, end_temperature):          #Fucntion to convert Fahrenheit to Celsius accepting starting and ending value of range [start, end]
```
Comment (kaustubhgupta) [inline Python/Fahrenheit_to_celsius.py]: Follow same instructions given for other PR regarding docstring and comments

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Python/Fahrenheit_to_celsius.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  33°F is 0.5556°C
+  34°F is 1.1111°C
+  35°F is 1.6667°C
+  '''
+  for fahrenheit in range (start_temperature, end_temperature + 1):
+    print(str(fahrenheit)+'°F', 'is', str(round((fahrenheit - 32) * 5/9, ndigits=4))+'°C')  #Celsius = (Fahrenheit - 32) * 5/9  (Formula to convert temperature from °Fehrenheit to °Celsius)
+    
+
+if __name__=="__main__":
+  fahrenheit_to_celsius(32, 212)                           #We pass the range of 32°F to 212°F which will output as 0°C and 100°C
```
Comment (kaustubhgupta) [inline Python/Fahrenheit_to_celsius.py]: The comments are still unformatted. Please perform a self-evaluation before requesting a review

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by kaustubhgupta** [comment]
@Addhyay  Did you read this message before requesting review? I am still seeing comments unformated

https://github.com/kaustubhgupta/Hacktoberfest2020_/pull/477#discussion_r508258165

**Pull request merged**